changelogs/current.yaml

```diff
@@ -34,5 +34,116 @@ removed_config_or_runtime:
 # *Normally occurs at the end of the* :ref:`deprecation period <deprecated>`
 
 new_features:
+- area: tls
```

Comment:
This delta looks like a bad merge.

Reply:
How can I fix this? I am just trying to get https://github.com/envoyproxy/envoy/pull/24302 merged into the v1.24 release so Istio can use it.

Reply:
delete the text blob below, and only leave this section:
```
- area: attributes
  change: |
    added :ref:`attributes <arch_overview_attributes>` for looking up xDS configuration information.
```